docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -164,6 +164,11 @@ belongs.
   `accuracy`:::
     (Optional, object) Accuracy of predictions (per-class and overall).
 
+  `auc_roc`:::
+    (Optional, object) The AUC ROC (area under the curve of the receiver
+    operating characteristic) score and optionally the curve. Default value is
```

Comment:
I think we should explain here that this calculates AUC ROC for a specific class. Plus, we need to think where we explain what happens to the docs that don't have the target class in the top classes.

Reply:
@przemekwitek Yes, I think it is a good idea to have`class_name` and `include_curve` as separated items under `auc_roc`.

Reply:
Done.